src/Environment.cpp

```diff
@@ -260,6 +260,21 @@ namespace geopm
         if (do_profile() && ret.empty()) {
             ret = std::string(program_invocation_name);
         }
+        else if (!ret.empty()) {
+            // Sanitize the input: No carriage returns nor double quotes
+            ret.erase(std::remove_if(ret.begin(), ret.end(),
```

Comment:
Maybe we could limit the warning messages to once per profile() call rather than once per bad character?  Maybe cache the start value, filter out bad values, and then print a warning if the string changed?  If we updated m_name_value_map we would be able to limit the warning message to be printed once per process...